src/components/modals/FormalQuestions.vue

```diff
@@ -953,6 +955,39 @@
               this.verificationData.type = "verification"
               this.verificationData.status = this.status
   
+              const isGenericSectionFilled = this.verificationData.locations !== undefined 
```

Comment:
You can do something like this

const requiredKeys = [
  'locations',
  'addressMatch',
  'customerDistance',
  'married',
  'nameOfSpouse',
  'customerChildren',
  'spouseSalary',
  'addressByGuarantor',
  'collateralPictures',
  'accommodationPicture',
  'hasBvn',
  'spouseNotGuarantor'
];

const isGenericSectionFilled = requiredKeys.every(key => this.verificationData[key] !== undefined);